source/manual/alerts/search-reindex-failed.html.md

```diff
@@ -0,0 +1,17 @@
+---
+owner_slack: "#2ndline"
+title: Search reindex failed
+parent: "/manual.html"
+layout: manual_layout
+section: Icinga alerts
+last_reviewed_on: 2017-11-23
+review_in: 12 months
+---
+
+The reindex task is run weekly on a Monday at 12pm on integration.  This is to ensure the process workes as
+expected when we need to run it in production.
+
+This task is will need to be run in production in the event of the elasticsearch schema being changed.
```

Comment:
Maybe merge this with the previous paragraph? On its own it sounds like it might be an instruction to 2nd line to run the task on production, but really it's just an explanation of why we do sometimes need to run it there.

Reply:
done